### ISSUE TITLE
Death without dying! - zC 0.0.11

### ISSUE THREAD

**Issue by Unyieldingvenom** [warezit/zitCraft#16]
Recently entered a twilight forest biome which caused the hunger debuff. Without any loss of health I suddeny got the message that I had died of starvation, all my items disappeared and a gravestone spawned but I did not die. Now I am constantly 'dying' and spawning gravestones but not actually dying. Eating does nothing to stop it and I cannot die by any other means, even being killed by another player just spawned a gravestone, said I died but I was still alive

**Comment by Rader2146** [comment]
Seems to happen to anyone who dies.  Same thing is happening to Zun.

**Comment by Unyieldingvenom** [comment]
Strange, does hunger kill immediately? I had 40 hearts of health it would have normally had to burn thru first but it didnt

**Comment by Rader2146** [comment]
Yes, going down to zero hunger does something like 600 damage: aka instant death.  The configs could be adjusted in the interim so that it isnt instant death.  That way If one is dying but not dying it will give a little time to eat before the death cycle continues.

**Comment by Unyieldingvenom** [comment]
Had to eat to stop death cycle, not easy but do-able :)

**Comment by warezit** [comment]
This is still a thing in zC 0.0.15.

Any advice on how to fix this is appreciated. I have skimmed thru Google briefly but not found anything related to this.

**Comment by Rader2146** [comment]
I have no idea where to start looking on this, but as a workaround for the meantime:

HungerOverhaul config option

```
# Amount of damage you take when hunger hits zero (vanilla default 2)
I:damageOnStarve=##
```

"##" = anything below 10 (5 hearts) would allow enough time to eat and end the death cycle.  HO default is 200.

**Comment by warezit** [comment]
Changing the HungerOverhaul.cfg from:
`I:damageOnStarve=200`

to:
`I:damageOnStarve=4`

Changes will take effect in zC 0.0.16

**Comment by warezit** [comment]
FINALLY fixed this in zC 2.0. Tested working/fine.
